gensrc/script/doris_builtins_functions.py

```diff
@@ -100,7 +104,7 @@
     [['element_at', '%element_extract%'], 'STRING', ['ARRAY_STRING', 'BIGINT'], 'ALWAYS_NULLABLE'],
 
     # map element
-    [['element_at', '%element_extract%'], 'INT', ['MAP_STRING_INT', 'STRING'], 'ALWAYS_NULLABLE'],
+    [['element_at', '%element_extract%'], 'V', ['MAP<K, V>', 'K'], 'ALWAYS_NULLABLE', ['K', 'V']],
 
```

Comment:
Great work! 
is this could apply on some function like array_sort, which input arg is all_array type, and is same as return type.
now we always register ARRAY_BOOLEAN........INT......., need more than 10 functions.

could be like this, only need one ?
`[['array_sort'], 'ARRAY<K>',   ['ARRAY<K>'], ''],`

Reply:
yes, only one is need for each array function. You should add [‘K’] in the last entry to specify which types are template type, just like C++ template declaration.

`[['array_sort'], 'ARRAY<K>',   ['ARRAY<K>'], '', ['K']]`

Reply:
`[['array_sort'], 'ARRAY<K>',   ['ARRAY<K>'], '', ['K']]` 
Thanks very mush! , use this way will be  easy to register array functions.

In addition, I want to know how to register if have a function eg:
it's input params is all array type, and return type is also all array type
it means, the input param  is `ARRAY<K>`, so the return type not just K,
like input is array<int>, but return will be array<boolean>, array<int>.....and so on